### PR TITLE
Add docs for sysctl tuning parameters

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Add docs for sysctl tuning parameters

## Motivation and Context
Borrowed from our performance documentation resources

## How to test this PR?
N/A

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [x] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#7240837b-4ea9-4d89-8e72-12f4980526a2) |
| mint-compression-dist-xl.sh | :x: [more...](#00a1d5a3-ac34-4b86-b832-80fedb6674c2) |
<a name="00a1d5a3-ac34-4b86-b832-80fedb6674c2"></a>
### 8749-b0b26eb/mint-compression-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31410
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 435841da5e6c:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 11 seconds
{"name": "awscli", "duration": 308973, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31410/awscli-mint-test-bucket-12144""}

Executed 0 out of 13 tests successfully.
```
<a name="7240837b-4ea9-4d89-8e72-12f4980526a2"></a>
### 8749-b0b26eb/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32623
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b80b3e4dbd52:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 23 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 1 minutes and 24 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 45 seconds
{
  "args": {},
  "duration": 79486,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#1112bb2c-0543-4daa-a711-03a16506c7e4) |
| mint-xl.sh | :x: [more...](#21ecb5dd-d71e-492a-935f-96e8320852fa) |
| mint-large-bucket.sh | :x: [more...](#e5e77f5c-e701-43c3-8862-72f826fb796c) |
| mint-worm.sh | :x: [more...](#6b3b3935-5f8b-4dd1-a789-79ee453d6d26) |
| mint-fs.sh | :x: [more...](#846c9982-d5fc-42bc-9eca-dfdd9aef6564) |
<a name="846c9982-d5fc-42bc-9eca-dfdd9aef6564"></a>
### 8749-cb7e033/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30967
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 3c69fb157894:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 12 seconds
{"name": "awscli", "duration": 309183, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:30967/awscli-mint-test-bucket-13944""}

Executed 0 out of 13 tests successfully.
```
<a name="21ecb5dd-d71e-492a-935f-96e8320852fa"></a>
### 8749-cb7e033/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31510
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 444ad54b8bf0:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 11 seconds
{"name": "awscli", "duration": 308437, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31510/awscli-mint-test-bucket-32233""}

Executed 0 out of 13 tests successfully.
```
<a name="6b3b3935-5f8b-4dd1-a789-79ee453d6d26"></a>
### 8749-cb7e033/mint-worm.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31208
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            worm
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 76f9742529cf:/mint/log /tmp/mint-logs'

(1/1) Running worm tests ... FAILED in 2 minutes and 1 seconds
{
  "alert": "",
  "args": {
    "bucketName": "bucket-cjkju93r0kxqcg690qzzfrdj3nl9uu71rwncebq3q8uxlhrjgxbh304",
    "expiry": 60000000000,
    "objectName": "testObject"
  },
  "duration": 120360,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.55:31208/bucket-cjkju93r0kxqcg690qzzfrdj3nl9uu71rwncebq3q8uxlhrjgxbh304: dial tcp 72.28.97.55:31208: i/o timeout",
  "function": "PutAndDelete",
  "message": "WORM_MODE ON  - CreateBucket Failed",
  "name": "test worm mode",
  "status": "FAIL"
}

Executed 0 out of 1 tests successfully.
```
<a name="e5e77f5c-e701-43c3-8862-72f826fb796c"></a>
### 8749-cb7e033/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:30670
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e938e4881c60:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 19 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 1 minutes and 6 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 5 seconds
{
  "args": {},
  "duration": 71764,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="1112bb2c-0543-4daa-a711-03a16506c7e4"></a>
### 8749-cb7e033/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:32480
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 58b321f3aab4:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 13 seconds
{
  "name": "awscli",
  "duration": 10397,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#8b3109bd-baab-44d5-9851-1475bc2a4c44) |
| mint-xl.sh | :x: [more...](#4d998430-4df9-4a2d-94d8-1140d8f3ad91) |
| mint-large-bucket.sh | :x: [more...](#6893624a-5216-41be-b74f-7702055aa9df) |
| mint-compression-fs.sh | :x: [more...](#bc04a0ec-047e-417f-8b43-702b85b8cae7) |
| mint-fs.sh | :x: [more...](#1933b8b8-005f-450c-a329-49a0d38e757d) |
<a name="1933b8b8-005f-450c-a329-49a0d38e757d"></a>
### 8749-2e2a8b1/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31070
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp cde249b76a0a:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 12 seconds
{"name": "awscli", "duration": 309126, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31070/awscli-mint-test-bucket-4306""}

Executed 0 out of 13 tests successfully.
```
<a name="6893624a-5216-41be-b74f-7702055aa9df"></a>
### 8749-2e2a8b1/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:32366
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e848c4a6f0b5:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 1 minutes and 23 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 5 seconds
(4/13) Running mc tests ... done in 40 seconds
(5/13) Running minio-go tests ... FAILED in 4 minutes and 6 seconds
{
  "args": {},
  "duration": 113936,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="4d998430-4df9-4a2d-94d8-1140d8f3ad91"></a>
### 8749-2e2a8b1/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:31969
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b6dd41a766f7:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 9 seconds
{
  "name": "awscli",
  "duration": 5967,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="8b3109bd-baab-44d5-9851-1475bc2a4c44"></a>
### 8749-2e2a8b1/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:32262
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e9e3ddb9ac33:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 13 seconds
{
  "name": "awscli",
  "duration": 11059,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="bc04a0ec-047e-417f-8b43-702b85b8cae7"></a>
### 8749-2e2a8b1/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31534
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4f30e74964ca:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 13 seconds
{"name": "awscli", "duration": 311077, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31534/awscli-mint-test-bucket-30543""}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#f9086a17-c8e6-4d97-848e-8f6edeb652a1) |
| mint-xl.sh | :x: [more...](#25ebed90-1861-4200-b987-9a5c08999ab2) |
| mint-large-bucket.sh | :x: [more...](#2191242c-dd4e-41b5-a6ee-e3c336e464b0) |
| mint-compression-fs.sh | :x: [more...](#47dfea89-cee6-4f1a-b072-bbe369dda0c5) |
| mint-gateway-nas.sh | :x: [more...](#f78b3d20-e764-4df5-8786-e9e778ddbc16) |
<a name="2191242c-dd4e-41b5-a6ee-e3c336e464b0"></a>
### 8749-b31df68/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32365
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp c84d58d7b679:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 16 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 6 seconds
(4/13) Running mc tests ... done in 1 minutes and 1 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 41 seconds
{
  "args": {},
  "duration": 92602,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="25ebed90-1861-4200-b987-9a5c08999ab2"></a>
### 8749-b31df68/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32342
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5a4504ff3e34:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 12 seconds
{
  "name": "awscli",
  "duration": 9655,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="f78b3d20-e764-4df5-8786-e9e778ddbc16"></a>
### 8749-b31df68/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32660
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e10f1838ddf0:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 11 seconds
{"name": "awscli", "duration": 308864, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:32660/awscli-mint-test-bucket-4560""}

Executed 0 out of 13 tests successfully.
```
<a name="f9086a17-c8e6-4d97-848e-8f6edeb652a1"></a>
### 8749-b31df68/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31281
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 33e62f409c2a:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 10 seconds
{"name": "awscli", "duration": 308556, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31281/awscli-mint-test-bucket-20213""}

Executed 0 out of 13 tests successfully.
```
<a name="47dfea89-cee6-4f1a-b072-bbe369dda0c5"></a>
### 8749-b31df68/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32075
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 14a38601362c:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 8 seconds
{"name": "awscli", "duration": 305392, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:32075/awscli-mint-test-bucket-18104""}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: docs/deployment/kernel-tuning/README.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ echo madvise | sudo tee /sys/kernel/mm/transparent_hugepage/enabled
 
 Also, set `transparent_hugepage=madvise` on your kernel command line (e.g. in /etc/default/grub) to persistently set this value.
 
+All these system level tunings are conveniently packaged in [shell script](https://github.com/minio/minio/blob/master/docsk/deployment/kernel-tuning/sysctl.sh). Please review the shell script for our recommendations.
```
Comment (Praveenrajmani) [inline docs/deployment/kernel-tuning/README.md]: @harshavardhana Can you fix the doc link - `https://github.com/minio/minio/blob/master/docs/deployment/kernel-tuning/disk-tuning.sh`

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#0f2f698a-73c8-44af-b583-8259ee907c15) |
| mint-gateway-nas.sh | :x: [more...](#e9d98fba-bf11-47a4-bf8e-951516546b66) |
<a name="e9d98fba-bf11-47a4-bf8e-951516546b66"></a>
### 8749-3bcd08e/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31244
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 411bc6c1906b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 12 seconds
{"name": "awscli", "duration": 309344, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31244/awscli-mint-test-bucket-3093""}

Executed 0 out of 13 tests successfully.
```
<a name="0f2f698a-73c8-44af-b583-8259ee907c15"></a>
### 8749-3bcd08e/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:31761
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 00fed3fb9ae3:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 21 seconds
(2/13) Running aws-sdk-java tests ... done in 2 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 1 minutes and 22 seconds
(5/13) Running minio-go tests ... FAILED in 2 minutes and 5 seconds
{
  "args": {},
  "duration": 808,
  "error": "The bucket you tried to delete is not empty",
  "function": "testStorageClassMetadataPutObject()",
  "message": "Cleanup failed",
  "name": "minio-go: testStorageClassMetadataPutObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: docs/deployment/kernel-tuning/README.md (hunk 1 above)
Comment (harshavardhana) [reply docs/deployment/kernel-tuning/README.md]:
```suggestion
All these system level tunings are conveniently packaged in [shell script](https://github.com/minio/minio/blob/master/docs/deployment/kernel-tuning/sysctl.sh). Please review the shell script for our recommendations.
```

**Review by nitisht — APPROVED** [review]

**Review by Praveenrajmani — APPROVED** [review]
The Travis failure looks irrelevant

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#64a0026e-e592-4548-a9ed-e93d4d8b722b) |
| mint-dist-xl.sh | :x: [more...](#d2189a1e-608b-4227-81b8-cf2aeca81d01) |
<a name="d2189a1e-608b-4227-81b8-cf2aeca81d01"></a>
### 8749-5e486d8/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:31253
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fd4910192d9b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 17 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 4 seconds
(4/13) Running mc tests ... done in 1 minutes and 27 seconds
(5/13) Running minio-go tests ... done in 1 minutes and 32 seconds
(6/13) Running minio-js tests ... FAILED in 7 seconds
{
  "name": "minio-js",
  "function": "initiateNewMultipartUpload(bucketName, objectName, metaData, cb)",
  "args": "bucketName:minio-js-test-48f13e42-4680-4b10-b3e9-233da1a12348, objectName:datafile-65-MB, metaData:[object Object]",
  "duration": 7,
  "status": "FAIL",
  "error": "S3Error: Please reduce your request at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:85:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:155:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:79:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:607:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:616:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:629:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:567:22) at IncomingMessage.onend (_stream_readable.js:727:10) at endReadableNT (_stream_readable.js:1215:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 5 out of 13 tests successfully.
```
<a name="64a0026e-e592-4548-a9ed-e93d4d8b722b"></a>
### 8749-5e486d8/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30734
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 984eb4d8c0ae:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 24 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 6 seconds
(4/13) Running mc tests ... done in 1 minutes and 32 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 23 seconds
{
  "args": {},
  "duration": 55162,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**